Contributing/How-to-Contribute-Code.md

```diff
@@ -0,0 +1,35 @@
+## Code Contributions
+
+We would like to thank you for your interest in contributing. You can find below how you can get started with knowing and contributing to keploy. The first few steps are always crucial but signify how you wish to involve with the community.
+
+1.  Get to know keploy:
+
+    -   Read the [Contributing guidelines](https://github.com/keploy/template/blob/main/CONTRIBUTING.md)
+    -   Read the [Code of Conduct](../CODE_OF_CONDUCT.md)
+    -   Review the [Coding Guidelines]() and the [Contribution Criteria](../CONTRIBUTING.md).
```

Comment:
Why is there an empty link?

Reply:
Because many folders and many things are not done